app/assets/javascripts/tinymce/plugins/alchemy_link/plugin.min.js

```diff
@@ -5,15 +5,15 @@ tinymce.PluginManager.add('alchemy_link', function(editor, url) {
     shortcut: 'Ctrl+K',
     stateSelector: 'a[href]',
     onclick: function () {
-      var link_object = {
+      var linkObject = {
         node: editor.selection.getNode(),
         bookmark: editor.selection.getBookmark(),
         selection: editor.selection,
         editor: editor
       };
-      var link_dialog = new Alchemy.LinkDialog(link_object);
+      var linkDialog = new Alchemy.LinkDialog(linkObject);
```

Comment:
@jorrizza please don't listen to @houndci-bot . We disabled JS linting in the `master` branch. Please don't change code, that you don't need to touch, thanks.